Makefile

```diff
@@ -20,6 +20,9 @@ system:
 	for extension in $(SystemJS); do \
 		echo $$extension"(System);" >> dist/$@.src.js; \
 	done
+	for extension in $(SystemJS); do \
+		echo "System._extensions.push("$$extension");" >> dist/$@.src.js; \
+	done
```

Comment:
We can do that, then the pattern would be for each extension to push itself into `._extensions`, do you think that is reasonable? It might be something where bugs don't suffer until you try to use a cloned loader... this might be acceptable, but I just wanted to bring it up.